playbooks/service-document-hosting-k8s.yaml

```diff
@@ -0,0 +1,11 @@
+- hosts: bridge.eco.tsi-dev.otc-service.com:!disabled
```

Comment:
let's call it service-docs.yaml. Inside we can have separate sections (similar to service-graphite) for handling k8 and vm deployments

Reply:
renamed